### DISCUSSION TITLE
How to build a XDA file independently?

### DISCUSSION THREAD

**Discussion by dd19990611** [idaholab/moose#20843] [Q&A Meshing] [answered]
I hope to couple another program with MOOSE and I choose XDA format to transfer meshes and fields. But I don't know rules of fields data in XDA files. It looks as if fields are written by vectors in XDA files. Fields distribution on meshes is involved with the sequence of variables in fields vector. 
I want to know rules of fields distribution in XDA files. If I write a field on a mesh, how to decide the sequence of variables in field vectors?

**Comment by GiudGiud** [comment, marked as answer]
@dd19990611 is this resolved? Issues and discussions are tracked differently, so we missed this.
XDA and XDR formats have their documentation here
[Description of the XDA/XDR mesh format used in libMeshhttps://sourceforge.net › mailman › attachment](https://sourceforge.net/p/libmesh/mailman/attachment/AD00A4C1B366594CB4B07B0A970306A90990AF91%40jsc-mail03.jsc.nasa.gov/1/)
